lib/rpc/SingleJsonRpcProvider.ts

```diff
@@ -76,8 +89,6 @@ export class SingleJsonRpcProvider extends StaticJsonRpcProvider {
         this.config.LATENCY_STAT_HISTORY_WINDOW_LENGTH_IN_S,
         log
       )
-      // Fire and forget. Won't check the sync result.
-      this.maybeSyncAndUpdateProviderState()
```

Comment:
This is to reduce the time to initialize lambda.
The first sync is actually optional.